h2o-r/h2o-package/R/communication.R

```diff
@@ -799,9 +799,42 @@ h2o.list_jobs <- function() {
   df
 }
 
-h2o.get_job <- function(job_key) {
+h2o.get_job <- function(job_key, jobPoolSucees = FALSE, jobIsRecoverable = FALSE) {
```

Comment:
spelling errors: pool -> poll; sucees -> success (I think "pool" is in other places as well)

Reply:
Thank you!